docs/adding-kubelet-args-to-existing-environment.md

```diff
@@ -0,0 +1,22 @@
+# Adding kubelet variables to existing environment
+
```

Comment:
I was thinking that you could add this in the migration folder for the next release.

Reply:
I missed that we had a migration folder. Will add it there instead.